src/app/cli/src/init/client.ml

```diff
@@ -1388,6 +1388,63 @@ let generate_libp2p_keypair =
                    ~metadata:[("err", `String (Error.to_string_hum e))] ;
                  exit 20 )) ))
 
+let whitelist_ip_flag =
+  Command.Param.(
+    flag "ip-address"
+      ~doc:"IP An IPv4 or IPv6 address for the client whitelist"
+      (required Cli_lib.Arg_type.ip_address))
+
+let whitelist_add =
+  let open Deferred.Let_syntax in
+  let open Daemon_rpcs in
+  Command.async ~summary:"Add an IP to the whitelist"
+    (Cli_lib.Background_daemon.rpc_init whitelist_ip_flag
+       ~f:(fun port whitelist_ip ->
+         let whitelist_ip_string = Unix.Inet_addr.to_string whitelist_ip in
+         match%map Client.dispatch Add_whitelist.rpc whitelist_ip port with
```

Comment:
Is it possible to use GraphQL rather than RPCs? We are trying to migrate all of our code to GraphQL.

Reply:
I will figure that out.